### ISSUE TITLE
iOS 13.7 Simulator getAvailablePurchases() throws NSInvalidArgumentException: +[NSInvocation _invocationWithMethodSignature:frame:] 

### ISSUE THREAD

**Issue by derRaab** [dooboolab/react-native-iap#2003]
**Description**

Calling `getAvailablePurchases()` on iOS 13.7 simulator using Storekit 1 throws this native exception: `[native] NSInvalidArgumentException: +[NSInvocation _invocationWithMethodSignature:frame:]: method signature argument cannot be nil`.

This is how it get's called:

```
    // NOTE: Keep react-native-iap@7 behaviour
    const alsoPublishToEventListener = true
    const automaticallyFinishRestoredTransactions = false

    const purchases = await getAvailablePurchases({
      alsoPublishToEventListener,
      automaticallyFinishRestoredTransactions,
    })
```

I also tried `automaticallyFinishRestoredTransactions = true` but got the same exception. Storekit2 seems to work just fine.

I would guess this also applies to real devices?

**Expected Behavior**

iOS Simulator should not crash.

**Environment:**

- react-native-iap: 11.0.3
- react-native: 0.68.3
- Platforms iOS 13.7 simulator

**To Reproduce**

1. Call `getAvailablePurchases()` within App on iOS 13.7 simulator.

**Comment by derRaab** [comment]
Additional information:

```
NSInvalidArgumentException: +[NSInvocation _invocationWithMethodSignature:frame:]: method signature argument cannot be nil
  File "RCTModuleMethod.mm", line 213, in -[RCTModuleMethod processMethodSignature]
  File "RCTModuleMethod.mm", line 522, in -[RCTModuleMethod invokeWithBridge:module:arguments:]
  File "RCTNativeModule.mm", line 183, in facebook::react::invokeInner
  File "RCTNativeModule.mm", line 104, in facebook::react::RCTNativeModule::invoke::lambda::operator()
  File "RCTNativeModule.mm", line 95, in facebook::react::RCTNativeModule::invoke
...
(10 additional frame(s) were not displayed)
```

<img width="583" alt="Bildschirmfoto 2022-09-27 um 09 51 35" src="https://user-images.githubusercontent.com/2079540/192466736-b557d858-c942-4873-a900-dba28da416c2.png">

**Comment by andresesfm** [comment]
Fix here: https://github.com/dooboolab/react-native-iap/pull/2007

**Comment by andresesfm** [comment]
This has been released as part of `11.0.4` Thank you for your patience @derRaab . Let me know if any other issues

**Comment by derRaab** [comment]
@andresesfm Thank you for your work! And - yes I will. :)
